atlas/AtlasMl/atlasml/ml/MLPipelines/PipelineWorkflows.py

```
@@ -163,29 +162,32 @@ def initial_cluster_pipeline(self, eps: float = 0.1, min_samples: int = 1, min_c
+                "description": competency["properties"]["description"],
                 "cluster_id": clusters[best_medoid_idx]["properties"]["cluster_id"],
                 "cluster_similarity_score": similarity_score[best_medoid_idx]
             }
             self.weaviate_client.update_property_by_id(CollectionNames.COMPETENCY.value, competency["id"], properties)
 
-        for index in range(len(texts)):
-            text_entry = texts[index]
-            competency = self.weaviate_client.get_embeddings_by_property(CollectionNames.COMPETENCY.value, "cluster_id", str(labels[index]))
+        for index in range(len(exercises)):
+            exercise = exercises[index]
+            cluster_center = self.weaviate_client.get_embeddings_by_property(CollectionNames.CLUSTERCENTER.value, "label_id", str(labels[index]))
```

Ruff (0.12.2)

Line too long (145 > 88)

(E501)